terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf

```
@@ -715,6 +715,25 @@ module "sles15sp5s390-minion" {
   install_salt_bundle = true
 }
 
+module "salt-migration-minion" {
```

What is the architecture?

---

It is a x86_64 SLES 15 SP5 minion  under the hood

---

Don't you want to include that information like in the others? It would make sense in case of questions about where you have taken some settings, such as `memory`. https://github.com/SUSE/susemanager-ci/blob/4c85dcf58bb32543893384986e2ca884339373f3/terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf#L338

---

comment or something like:
```suggestion
module "salt-migration-sles15sp5-minion" {
```
That's optional, close this conversation when satisfied.